### PR TITLE
 [IMP]account:Some screens are simplify.

### PR DESCRIPTION
**Pull request by bpa-odoo**

### CODE REVIEW

**Comment by rim-odoo** [comment]
Please submit PR against odoo/odoo

**Comment by mdi-odoo** [comment]
Hello @rim,

This task is related to Accounting v9.0 and also lots of models are changed and some models are removed like 'account.period', ...etc.
I hope you can understand this.

Thanks,

**Comment by rim-odoo** [comment]
Ok sorry, I read to fast

File: addons/account/account.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ def _get_company_currency(self):
         help="Account Type is used for information purpose, to generate "\
         "country-specific legal reports, and set the rules to close a fiscal year and generate opening entries.")
     child_consol_ids = fields.Many2many('account.account', 'account_account_consol_rel', 'child_id', 'parent_id', string='Consolidated Children', domain=[('deprecated', '=', False)])
-    last_time_entries_checked = fields.Datetime(string='Latest Manual Reconciliation Date', readonly=True, copy=False,
+    last_time_entries_checked = fields.Datetime(string='Reconciliation Date', readonly=True, copy=False,
```
Comment (Whisno) [inline addons/account/account.py]: Why this change ?
